projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-quote.ts

```diff
@@ -866,6 +868,34 @@ export function navigateToQuoteListFromMyAccount() {
     });
 }
 
+/**
+ * Navigates to the vendor quote list via my account.
+ */
+export function navigateToVendorQuoteListFromMyAccount() {
```

Comment:
MAJOR: Please place the help functions that we have created inside helper/vendor/cpd (create new cpq folder), as we are placing out test under e2e/vendor